.github/workflows/scripts/prs/checkTargetBranchLabel.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
 const vBranchRegex = /^v\d{1,3}\.x$/;
-const transferLabels = ['cherry-pick'];
+// const transferLabels = ['cherry-pick'];
```

Comment:
can't we just remove? Or does this have a function?

```suggestion
// const transferLabels = ['cherry-pick'];
```